content/post/practical-stegen.Rmd

```diff
@@ -0,0 +1,825 @@
+stool samples.
+
+The Freiburg health office sent a team to investigate the kitchen of the
+caterer. Food preparation procedures were reviewed. Food samples, except
+tiramisu (none was left over), were sent to the laboratory of Freiburg
+University. Microbiological analyses were performed on samples of the following:
+brown chocolate mousse, caramel cream, remoulade sauce, yoghurt dill sauce, and
+10 raw eggs.
+
+The Freiburg health office requested help from the RKI in the investigation to
+assess the magnitude of the outbreak and identify potential vehicle(s) and risk
+factors for transmission in order to better control the outbreak.
+
+
+
+## The epidemiological study
+
+**Case definition**: cases were defined as any person who had attended the party
+at St Sebastian High School and who suffered from *diarrhoea* (min. 3 loose
+stool for 24 hours) between 27 June and 29 June 1998, or from at least three of
+the following symptoms: *vomiting*, *fever over 38.5° C*, *nausea*, 
+*abdominal pain*, *headache*.
+
+Students from both schools attending the party were asked through phone
+interviews to provide names of persons who attended the party.
+
+Overall, 291 responded to enquiries and 103 cases were identified. The linelist
+analysed in this case study was built from these 291 responses.
+
+
+## This case study
+
+In this case study, we will take you through the analysis of this epidemic. This
+will be the occasion to illustrate more generally useful practices for data
+analysis using **R**, including:
+
+- how to read data from Excel
+- how to explore data using tables and summaries
+- how to clean data
+- how to make graphics to describe the data
+- how to test if specific food items are linked to the disease
+
+
+
+
+
+# Initial data processing
+
+## Loading required packages
+
+The following packages will be used in the case study:
+
+- `here`: to find the path to data or script files
+- `readxl`: to read Excel spreadsheets into **R**
+- `incidence`: to build epicurves
+
+To avoid having to specify the name of the package whenever we use a function
+using the syntax `package_name::function_name()`, we load these packages using
+`library`:
+
+```{r stegen_libraries, warning=FALSE, message=FALSE}
+library(here)      # find data/script files
+library(readxl)    # read xlsx files
+library(incidence) # make epicurves
+library(epitrix)   # clean labels and variables
+library(dplyr)     # general data handling
+library(ggplot2)   # advanced graphics
+```
+
+Note that you will get an error if the packages have not been installed on your
+system. To install them (you only need to do this once!), type:
+
+```{r stegen_install_libraries, eval = FALSE}
+install.packages("here")
+install.packages("readxl")
+install.packages("incidence")
+install.packages("epitrix")
+install.packages("dplyr")
+install.packages("ggplot2")
+```
+
+
+Loading these packages makes all functions implemented by the packages
```

Comment:
I would leave that in extra comments, otherwise people are hung up on this. In Sofia people had quite a few questions about this. It is an interesting point to make, in terms of training, but I would argue maybe not a priority for beginners?